lib/openstudio-standards/standards/Standards.ThermalZone.rb

```diff
@@ -1375,6 +1375,16 @@ def thermal_zone_conditioning_category(thermal_zone, climate_zone)
   # @return [Double] the design heating supply temperature, in C
   # @todo Exception: 17F delta-T for labs
   def thermal_zone_prm_baseline_heating_design_supply_temperature(thermal_zone)
+
+    if /prm/i =~ template # avoid affecting previous PRM tests
+      # 90.1 Appendix G G3.1.2.8.2
+      thermal_zone.equipment.each do |eqt|
+        if eqt.to_ZoneHVACUnitHeater.is_initialized
+          return OpenStudio.convert(105, 'F', 'C').get
+        end
+      end
+    end
+
```

Comment:
Sorry if this seems like nitpicking ... I think this should go in a method under the "90.1-PRM" or "90.1-PRM-2019" class. I feel like we should try to avoid the `if /prm/i =~ template` as much as possible. Of course, we would need the same function in Standards but there it could do nothing, just return `false` maybe.